examples/Responsive.js

```diff
@@ -10,6 +21,7 @@ export default class Responsive extends Component {
       slidesToShow: 4,
       slidesToScroll: 4,
       initialSlide: 0,
+      unslick: UnSlick,
```

Comment:
@blairanderson  Can you give me an use where this feature feature will be useful ?

If you want to show the slide vertically on the smaller devices, we already have vertical feature. Check this example
https://github.com/akiran/react-slick/blob/master/examples/VerticalMode.js
You can set vertical in responsive settings.

Let me know if this helps